bin/ldap_query.py

```diff
@@ -0,0 +1,26 @@
+from ldap3 import Server, Connection, ALL
+from ldap3 import MODIFY_REPLACE, ALL_ATTRIBUTES
+
+server = Server('ldap://${ip}', get_info=ALL)
+conn = Connection(server, user='svc_adds_sync', password='${password}')
+if not conn.bind():
+  raise ValueError("Error connecting to LDAP server %s ", conn.result)
+
+
+#search_filter = '(&(objectClass=person)(distinguishedName=CN=testUser,OU=Privileged,OU=Users,OU=Accounts,DC=clouddev,DC=corporate,DC=t-mobile,DC=com))'
+search_filter = '(objectClass=person)'
```

Comment:
Above this line you should add ```# search_filter = '(objectClass=group)'``` as this will be the code for group searching.   Keep it commented out is fine.